src/resolvers/registries/npm-resolver.js

```diff
@@ -24,6 +24,10 @@ export default class NpmResolver extends RegistryResolver {
   static registry = 'npm';
 
   static async findVersionInRegistryResponse(config: Config, range: string, body: RegistryResponse): Promise<Manifest> {
+    if (!body['dist-tags']) {
```

Comment:
this check feels very loose. I wonder if we can do better or if this is the best we can do...